drivers/video/video_stm32_dcmi.c

```diff
@@ -308,12 +308,17 @@ static int video_stm32_dcmi_enqueue(const struct device *dev,
 				  struct video_buffer *vbuf)
 {
 	struct video_stm32_dcmi_data *data = dev->data;
+	const uint32_t buffer_size = data->pitch * data->height;
 
 	if (ep != VIDEO_EP_OUT) {
 		return -EINVAL;
 	}
 
-	vbuf->bytesused = data->pitch * data->height;
+	if (vbuf->size < buffer_size) {
```

Comment:
In general, the variable part is on the left side, while the limit/fixed part is on the right side.
It would hence be more readable as follows:
```suggestion
if (buffer_size > vbuf->size) {
```

Reply:
Sounds good to me. Revising now.